README.md

```diff
@@ -1,32 +1,158 @@
-# python-flask-template
+OpenFaaS Python Flask Templates
+=============================================
 
-Python OpenFaaS template with Flask
+The Python Flask templates make use of the incubator project [of-watchdog](https://github.com/openfaas-incubator/of-watchdog) to handle higher throughput than the [classic watchdog](https://github.com/openfaas/faas/tree/master/watchdog) due to the process being kept warm.
```

Comment:
I think this is technically correct but confusing / too-much-info for the people who may read it.

> to handle higher throughput than the [classic watchdog](https://github.com/openfaas/faas/tree/master/watchdog) due to the process being kept warm.